src/gui/transferlistfilterswidget.h

```diff
@@ -70,7 +65,7 @@ private slots:
     virtual void torrentAboutToBeDeleted(BitTorrent::TorrentHandle *const) = 0;
 };
 
-class StatusFiltersWidget: public FiltersBase
+class StatusFiltersWidget : public FiltersBase
```

Comment:
It seems like plural form is inappropriate in these class names. It should be StatusFilterWidget and so on.